localization/ekf_localizer/include/ekf_localizer/warning_message.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef EKF_LOCALIZER__WARNING_MESSAGE_HPP_
+#define EKF_LOCALIZER__WARNING_MESSAGE_HPP_
+
+#include <string>
+
+std::string poseDelayStepWarningMessage(
+  const double delay_step, const double extend_state_step, const double ekf_dt);
```

Comment:
It's better to make `extended_state_step` as `int`, given that it is defined so in `hyper_parameters.hpp`.
```suggestion
  const double delay_step, const int extend_state_step, const double ekf_dt);
```